similarity_search/test/test_distfunc.cc

```diff
@@ -785,8 +785,8 @@ bool TestSparseCosineSimilarityAgree(const string& dataFile, size_t N, size_t Re
 
     bool bug = false;
 
-    float maxRelDiff = 1e-6f;
-    float maxAbsDiff = 1e-6f;
+    float maxRelDiff = 1e-5f;
+    float maxAbsDiff = 1e-5f;
```

Comment:
Yes, there is, in fact a ticket to enable setting a seed value, which can be also specified when initializing the library. This should also greatly help to "stabilize" the integration test.